src/Changelog/ReleaseChangelog.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\AutomaticReleases\Changelog;
+
+use Laminas\AutomaticReleases\Git\Value\BranchName;
+use Laminas\AutomaticReleases\Git\Value\SemVerVersion;
+
+interface ReleaseChangelog
```

Comment:
This name is not clear: is it  supposed to be "commit release changelog" perhaps? Since its signature is `: void`, I suppose it **does** something

Reply:
Yeah, `CommitReleaseChangelog` makes sense - I was thinking "release" as a verb here, but we also have a concept of "a release" throughout. If that name makes sense, I'll update.